path-timestamp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Expose timestamp of given path
+#
+# Usage: add this to crontab:
+#
+# */5 * * * * prometheus path-timestamp.sh /var/lib/prometheus | sponge /var/lib/node_exporter/path-timestamp.prom
+#
+# Author: Antoine Beaupré <anarcat@debian.org>
+
+echo "# HELP node_path_modification_timestamp_seconds Last change timestamp"
+echo "# TYPE node_path_modification_timestamp_seconds gauge"
+for path in "$@"; do
+    printf 'node_path_modification_timestamp_seconds{path="%s"}' "$path"
```

Comment:
```suggestion
    printf 'node_path_modification_timestamp_seconds{path="%s"} ' "$path"
```

there's currently no space between the list of labels and the value

Reply:
fixed